src/rules/prefer-use-deep-compare-memo.ts

```
@@ -0,0 +1,402 @@
+            // Ensure import exists
+            fixes.push(...ensureDeepCompareImportFixes(context, fixer));
+
+            return fixes;
+          },
+        });
+      },
+    };
+  },
+});
+
+export default preferUseDeepCompareMemo;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)